src/decider/agent_tests/Functional/schedulerTest.php

```diff
@@ -626,7 +626,7 @@ private function runnerShouldMakeDecisionAsWipIfUnhandledScannerEvent($runner)
     /* insert NoLicenseKnown decisions */
     $this->dbManager->queryOnce("INSERT INTO clearing_decision (clearing_decision_pk, uploadtree_fk, pfile_fk, user_fk, group_fk, decision_type, scope, date_added)"
             . " VALUES (2, $itemId, $pfile, $userId, $groupId, ".DecisionTypes::IDENTIFIED.", ".DecisionScopes::ITEM.", '2015-05-04 11:43:18.276425+02')");
-    $isWipBeforeDecider = $this->clearingDao->isDecisionWip($itemId, $groupId);
+    $isWipBeforeDecider = $this->clearingDao->isDecisionCheck($itemId, $groupId, 'WIP');
```

Comment:
How about using the class constants?
```suggestion
    $isWipBeforeDecider = $this->clearingDao->isDecisionCheck($itemId, $groupId, DecisionTypes::WIP);
```